### ISSUE TITLE
openwrt version

### ISSUE THREAD

**Issue by usrhere** [wknapik/vpnfailsafe#27]
Looks like a great script, would be nice to have a version of it for openwrt. I think it might need a bit of modification as openwrt is quite minimal and doesnt have getent. Please consider it :) thanks!

**Comment by wknapik** [comment]
Hi @juoza. Thanks for the suggestion. I never thought of it, but it sounds like a nice idea. I don't currently own an OpenWRT-based device, but I could just use a VM, I guess. I'm between holidays right now, so I won't look at this for a while, but I will get around to it eventually.

**Comment by wknapik** [comment]
Depends on #32.
